contracts/lib/uniswap/uniswap/UniswapExchange.sol

```diff
@@ -0,0 +1,613 @@
+pragma solidity ^0.5.0;
```

Comment:
I'm just wondering; if we're only using these contracts in the tests (I assume so, since we already have the interface versions), can we put these in a `test/` dir?